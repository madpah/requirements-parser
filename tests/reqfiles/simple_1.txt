z3c.checkversions==0.4.1
MyPackage
Framework==0.9.4
Library>=0.2
