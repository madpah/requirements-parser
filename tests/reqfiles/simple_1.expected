[
 {
  "extras": [], 
  "name": "z3c.checkversions", 
  "specs": [
   [
    "==", 
    "0.4.1"
   ]
  ]
 }, 
 {
  "extras": [], 
  "name": "MyPackage", 
  "specs": []
 }, 
 {
  "extras": [], 
  "name": "Framework", 
  "specs": [
   [
    "==", 
    "0.9.4"
   ]
  ]
 }, 
 {
  "extras": [], 
  "name": "Library", 
  "specs": [
   [
    ">=", 
    "0.2"
   ]
  ]
 }
]